core/notification/src/main/java/edu/stanford/spezi/core/notification/di/NotificationModule.kt

```diff
@@ -0,0 +1,37 @@
+package edu.stanford.spezi.core.notification.di
+
+import com.google.firebase.messaging.FirebaseMessaging
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import edu.stanford.spezi.core.notification.fcm.DefaultMessageTokenService
+import edu.stanford.spezi.core.notification.fcm.MessageTokenService
+import edu.stanford.spezi.core.notification.notifier.NotificationNotifier
+import edu.stanford.spezi.core.notification.notifier.SystemTrayNotificationNotifier
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+internal class NotificationModule {
+    @Provides
+    @Singleton
+    internal fun provideFirebaseMessaging(): FirebaseMessaging =
+        FirebaseMessaging.getInstance().apply {
```

Comment:
can we remove the empty apply block or are you planning any further configuration?

Reply:
i thought i could add a debug emulator setting similar to the other firebase components but the component doesn't seem to support it and then i must have forgotten the apply block. i removed it